src/App.js

```diff
@@ -1,11 +1,23 @@
-import logo from './logo.svg';
+/* eslint-disable react/jsx-pascal-case */
 import './App.css';
-
+import { useState ,createContext} from 'react';
+import Main_head from './components/Main_head';
+import Header from './components/Header'
+import Proudects from './components/Proudects';
+import Footer from './components/Footer';
+export const ProductContxt = createContext(0)
 function App() {
+  const [clicked,setClicked] = useState(0)
   return (
-    <div className="App">
-    </div>
+    <div className="App" >
+      <ProductContxt.Provider value={{ clicked, setClicked }}>
+        <Main_head />
```

Comment:
Be consistent in naming components so rename it to <MainHead/>

Reply:
done